lib/galaxy/model/__init__.py

```diff
@@ -144,6 +145,7 @@
 if TYPE_CHECKING:
     class _HasTable:
         table: Table
+        __table__: Table
```

Comment:
HDA, LDDA, Dataset don't have a `__table__` attribute (at least not yet  - I'm working on it, so they will once they are declaratively mapped). But since these live in `__annotations__`, I guess that doesn't matter.